media-sound/clementine/clementine-1.4.1~git.recipe

```diff
@@ -112,8 +112,10 @@ BUILD_REQUIRES="
 BUILD_PREREQUIRES="
 	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
+	cmd:git
```

Comment:
Adding cmd:git is not really needed in general, in this case it doesn't change anything also, so not needed here.